images/Dockerfile.nvidia

```diff
@@ -55,6 +55,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
     file \
     fuse \
     gawk \
+    gdisk \
```

Comment:
gdisk provides `sgdisk` ?

Reply:
yep!
`fdisk` provides `sfdisk`, and `gdisk` provides `sgdisk` :rofl: